### ISSUE TITLE
Random style or ALL Styles Batch

### ISSUE THREAD

**Issue by simartem** [yolain/ComfyUI-Easy-Use#377]

**Comment by yolain** [comment]
Are you referring to the styles selector node? It doesn't look like it's going to be easy to implement.

**Comment by simartem** [comment]
> Are you referring to the styles selector node? It doesn't look like it's going to be easy to implement.

yes refer to styles selector. sometimes we cant determine which style works the best, so it should be nice to have these functions, at least the random would help in a batch prompt generation.. thank you in advance
